### PR TITLE
LPS-146463 Add implementation of isApplicable method

### PR DESCRIPTION
**Pull request by seltong**

### CODE REVIEW

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @seltong, can you please check if these failures are related to your changes? Thanks

**Comment by liferay-forms** [comment]
Hey @seltong, can you please check if these failures are related to your changes? Thanks

**Comment by seltong** [comment]
Hi, @liferay-forms. I will take a look at this. Thanks

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @seltong, can you please check if these failures are related to your changes? Thanks

**Comment by seltong** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @seltong, can you please check if these failures are related to your changes? Thanks

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
Just started reviewing :)

**Comment by MarinhoFeliphe** [comment]
Hi @seltong,

join the repeated code in just one([for example](https://github.com/seltong/liferay-portal/pull/42/commits/241e44883f953ca39654ac7e390fcdd9c677186d)) and rename `getUsersThatCanSeeNotifications` to `getUsersAllowedToViewNotifications`

Thank you!

**Comment by seltong** [comment]
Hi @MarinhoFeliphe ,

The requested changes was done.

Thank you,
Selton

File: portal-kernel/src/com/liferay/portal/kernel/workflow/WorkflowConstants.java [hunk 1]
```diff
@@ -107,6 +109,8 @@
 
 	public static final int TYPE_COMPLETE = 10001;
 
+	public static final String VIEW_TASK_NOTIFICATION = "VIEW_NOTIFICATION";
```
Comment (MarinhoFeliphe) [inline portal-kernel/src/com/liferay/portal/kernel/workflow/WorkflowConstants.java:112]: Hi @seltong,

Please, rename the value of this variable to VIEW_TASK_NOTIFICATION

Thanks!

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]
Hi @seltong,

Could you please split the first commit for each modified file?

Thank you!

**Comment by seltong** [comment]
ci:forward

**Pull request closed without merging**